WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ProductRowViewModel.swift

```diff
@@ -121,6 +140,22 @@ final class ProductRowViewModel: ObservableObject, Identifiable, Equatable {
         let unformattedPrice = price.isNotEmpty ? price : "0"
         return currencyFormatter.formatAmount(unformattedPrice)
     }
+
+    /// Increment the product quantity.
+    ///
+    func incrementQuantity() {
+        quantity += 1
+    }
+
+    /// Decrement the product quantity.
+    /// This must check that the quantity is above the minimum value before decrementing it.
+    ///
+    func decrementQuantity() {
+        guard shouldDisableQuantityDecrementer == false else {
```

Comment:
Super nit:
```suggestion
        guard !shouldDisableQuantityDecrementer else {
```

Reply:
Another nit:
Although the logic is the same, I think it reads a bit weird asking if a view component is disabled before proceeding with a view model action. 

I think the explicit condition  like  `quantity > 1` express the intention better 😅

Reply:
> I think the explicit condition like quantity > 1 express the intention better 😅

Makes sense! I think my intention here was to reuse the same check, since both the decrement action and the view component are based on the same minimum value. But I agree that `quantity > 1` is clearer here.

Reply:
I decided to explicitly set a `minimumQuantity` and use that for both checks in e79dbf9, which I think makes the intention clearer.